sellotape/sellotape_dj/sellotape_dj/routing.py

```diff
@@ -0,0 +1,12 @@
+from channels.auth import AuthMiddlewareStack
```

Comment:
Note, you fail the tests:

ModuleNotFoundError: No module named 'channels'
Error: Process completed with exit code 1.

Please open a PR that adds the 'channels' module before pushing this PR, so we can see that the tests check out

Reply:
I've done that, tests still going wrong.
can you please check again?